core/src/main/java/com/graphhopper/routing/subnetwork/PrepareRoutingSubnetworks.java

```diff
@@ -56,155 +68,118 @@ public PrepareRoutingSubnetworks(GraphHopperStorage ghStorage, List<PrepareJob>
     }
 
     /**
-     * The subnetwork removal removes components with less than {@link #minNetworkSize} nodes from the graph if it is
-     * run node-based. For edge-based subnetwork removal it removes components with less than 2*{@link #minNetworkSize}
-     * (directed) edges.
+     * All components of the graph with less than 2*{@link #minNetworkSize} directed edges (edge keys) will be marked
+     * as subnetworks. The biggest component will never be marked as subnetwork, even when it is below this size.
```

Comment:
All subnetwork preparation is now edge-based so no need to talk about node-based anymore.